src/sarif_converter.ts

```diff
@@ -45,22 +45,18 @@ const getTools = (analysisResults: IAnalysisResult, suggestions: ISarifSuggestio
   const rules = [];
   let ruleIndex = 0;
   for (const [suggestionName, suggestion] of <[string, ISuggestion][]>Object.entries(analysisResults.suggestions)) {
-    let severity;
-    const severityNum: number = suggestion.severity;
-    if (severityNum > 0 && severityNum <= 3) {
-      severity = {
-        3: 'error',
-        2: 'warning',
-        1: 'note',
-      }[severityNum];
-    }
+    const severity = <Result.level>{
+      1: 'note',
+      2: 'warning',
+      3: 'error',
+    }[suggestion.severity];
 
     const suggestionId = suggestion.id;
     const rule = {
       id: suggestionId,
       name: suggestion.rule,
       shortDescription: {
-        text: suggestion.message,
+        text: suggestion.title || suggestion.rule,
       },
       fullDescription: {
```

Comment:
Sorry can you please remove this (`fullDescription) too? (`suggestion.message` again)

Reply:
done